Makefile

```diff
@@ -18,9 +18,6 @@
 STAGINGIMAGE=${GCE_PD_CSI_STAGING_IMAGE}
 STAGINGVERSION=${GCE_PD_CSI_STAGING_VERSION}
 
-PRODIMAGE=gcr.io/gke-release/gcp-compute-persistent-disk-csi-driver
```

Comment:
Why were these lines removed?

Reply:
we aren't building a prod image from the makefile ever (we use the internal release flows). I wanted to keep them around for "documentation" purposes for a while, but now our actual documentation captures this same information so I think its reasonable to remove